### PR TITLE
Hurricane Irene Edited

### PR DESCRIPTION
**Pull request by suhsha21**

### CODE REVIEW

File: irene/setrun.py [hunk 1]
```diff
@@ -400,88 +386,84 @@ def setgeo(rundata):
...
-    topo_data.topofiles.append([4, 1, 6, days2seconds(-1.45), days2seconds(-0.5),os.path.join(topo_path,'nc40x75_73x75.nc')])
-
-    # == setqinit.data values ==
-    rundata.qinit_data.qinit_type = 0
-    rundata.qinit_data.qinitfiles = []
-    # for qinit perturbations, append lines of the form: (<= 1 allowed for now!)
-    #   [minlev, maxlev, fname]
+    # See regions for control over these regions, need better bathy data for
+    # the smaller domains
+    topo_path = os.path.join(datadir, 'IreneTopo2.nc')
```
Comment (mandli) [inline irene/setrun.py:391]: Should remove the code that converts the NetCDF file

File: irene/setrun.py [hunk 2]
```diff
@@ -7,22 +7,32 @@
...
+def days2seconds(days):
+    return days * 60.0**2 * 24.0
 
-#                           days   s/hour    hours/day            
-days2seconds = lambda days: days * 60.0**2 * 24.0
-seconds2days = lambda seconds: seconds / (60.0**2 * 24.0)
 
-#------------------------------
+# Scratch directory for storing topo and dtopo files:
+scratch_dir = os.path.join(os.environ["CLAW"], 'geoclaw', 'scratch')
```
Comment (mandli) [inline irene/setrun.py:31]: Should not use the scratch directory for apps

File: irene/setrun.py [hunk 3]
```diff
@@ -400,88 +386,84 @@ def setgeo(rundata):
...
-    rundata.qinit_data.qinitfiles = []
-    # for qinit perturbations, append lines of the form: (<= 1 allowed for now!)
-    #   [minlev, maxlev, fname]
+    # See regions for control over these regions, need better bathy data for
+    # the smaller domains
+    topo_path = os.path.join(datadir, 'IreneTopo2.nc')
+    irenetopo = topo.read_netcdf(topo_path,coarsen=2, verbose=True)
+    irenetopo.write(os.path.join(datadir, 'irenetopo.tt3'), topo_type=3, Z_format='%15.7e')	    
+    irene_path=os.path.join(datadir, 'irenetopo.tt3')
+    topo_data.topofiles.append([3, 1, 5, rundata.clawdata.t0,
```
Comment (mandli) [inline irene/setrun.py:395]: This needs to be updated also for clawpack 5.8.0.

File: irene/setrun.py [hunk 4]
```diff
@@ -400,88 +386,84 @@ def setgeo(rundata):
...
+    # Storm parameters - Parameterized storm (Holland 1980)
+    data.storm_specification_type = 'holland80'  # (type 1)
+    data.storm_file = os.path.expandvars(os.path.join(os.getcwd(),
+                                         'irene.storm'))
 
-    return data
+    # Convert ATCF data to GeoClaw format
+    clawutil.data.get_remote_file(
+                   "http://ftp.nhc.noaa.gov/atcf/archive/2011/bal092011.dat.gz")
+    atcf_path = os.path.join(scratch_dir, "bal092011.dat")
```
Comment (mandli) [inline irene/setrun.py:429]: Also probably should keep this to the local directory rather than the scratch.

**Review by mandli — CHANGES_REQUESTED** [review]
Most we just need to adjust paths.

**Comment by suhsha21** [comment]
I fixed the paths and made the suggested changes. Thank you.

**Pull request merged**